<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="tracer">
    <xacro:property name="pi" value="3.1415926"/>
    <xacro:property name="chasis_radius" value="0.4"/>
    <xacro:property name="chasis_length" value="0.4"/>
    <xacro:property name="chasis_mass" value="10.0"/>
    
    <xacro:property name="wheel_radius" value="0.12"/>
    <xacro:property name="wheel_length" value="0.05"/>
    <xacro:property name="wheel_dist" value="0.6"/>
    <xacro:property name="wheel_mass" value="10.0"/>
    
    <xacro:property name="sphere_radius" value="0.025"/>
    <xacro:property name="sphere_mass" value="0.1"/>
    
    <xacro:property name="lidar_radius" value="0.1"/>
    <xacro:property name="lidar_length" value="0.1"/>
    <xacro:property name="lidar_mass" value="0.1"/>
    
    <xacro:property name="camera_size_x" value="0.05"/>
    <xacro:property name="camera_size_y" value="0.2"/>
    <xacro:property name="camera_size_z" value="0.1"/>
    <xacro:property name="camera_mass" value="0.1"/>
    
</robot>





